scripts/RoboFile.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * This is project's console commands configuration for Robo task runner.
+ *
+ * @see http://robo.li/
+ */
+class RoboFile extends \Robo\Tasks
+{
+  const PROJECT_ROOT = __DIR__ . '/../';
```

Comment:
I suspect PHP 5.5 is going to freak out at this concatenation outside a function.